addons/l10n_hn/__manifest__.py

```diff
@@ -19,7 +19,7 @@
 moneda Lempira. -- Adds accounting chart for Honduras. It also includes taxes
 and the Lempira currency.""",
     'author': 'Salvatore Josue Trimarchi Pinto',
-    'website': 'http://bacgroup.net',
+    'website': 'https://www.facebook.com/salvatore.py',
```

Comment:
Ideally, this would not be just a personal page as it is under "Learn More" of the module and should tell something about it.